src/main/java/fr/moribus/imageonmap/commands/maptool/GetCommand.java

```diff
@@ -49,11 +53,53 @@
 public class GetCommand extends IoMCommand {
     @Override
     protected void run() throws CommandException {
-        Player player = playerSender();
-        if (getMapFromArgs().give(player)) {
-            info(I.t("The requested map was too big to fit in your inventory."));
-            info(I.t("Use '/maptool getremaining' to get the remaining maps."));
+        ArrayList<String> arguments = getArgs();
+
+        if (arguments.size() > 2) {
+            warning(I.t("Too many parameters! Usage: /maptool get [playername]:<mapname>"));
```

Comment:
Plus usage dans `@CommandInfo`
```suggestion
            throwInvalidArgument(I.t("Too many parameters!"));
```